study.md

```diff
@@ -147,5 +160,5 @@ using `hank` that equals "My name is Hank Hill and I'm a propane and propane
 accesories salesman that enjoys drinking beer.".
 
 ```javascript
-// your answer here
+console.log("My name is " + hank['given-name'] + " " + hank.family_name +  " and I'm a " + hank.occupation + " that enjoys " + hank.hobbies[0] + ".")
```

Comment:
This will print the correct output, but we want to use template literals and not string concatenation.

```javascript
const string = `My name is ${hank['given-name']} ${hank.family_name} and I'm a ${hank.occupation} that enjoys ${hank.hobbies[0]}.`;
```